arm-network/2017-06-01/swagger/applicationGateway.json

```diff
@@ -390,7 +390,7 @@
         "tags": [
           "ApplicationGateways"
         ],
-        "operationId": "ApplicationGateways_ListAvailableSslOptions",
+        "operationId": "ApplicationGateways_AvailableSslOptions",
```

Comment:
I don't see AvailableSslOptions operation in 2017-03-01, it it's being introduced in this API, it'd be better to follow the naming convention. Did this operation exist before? @akshaysngupta what may break if this operation is named as you originally proposed in this PR?